changelog/add-8686-track-events-payment-activity-widget

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
```

Comment:
I believe if you want to just add a comment that won't result a changelog entry, there shouldn't be an empty line here.

Search for `Comment:` in `changelog/` to see other 'comment' changelogs.